lib/components/jpegdepay/jpegdepay-component.js

```diff
@@ -0,0 +1,57 @@
+const {Transform} = require('stream')
+const {Rtp} = require('../../utils/protocols')
+const {SDP, RTP, JPEG} = require('../messageTypes')
+const Component = require('../component')
+const jpegDepayFactory = require('./jpegdepay')
+
+class JPEGDepayComponent extends Component {
+  constructor () {
+    let payloadType
+    let packets = []
+    let jpegDepay
+
+    const incoming = new Transform({
+      objectMode: true,
+      transform (msg, encoding, callback) {
+        if (msg.type === SDP) {
+          payloadType = Number(msg.sdp.media[0].rtpmap.payloadType)
```

Comment:
Can we be sure this index is the correct one?

Reply:
No, this was bad. Fixed it properly now with JPEG media detection.